tokio/src/runtime/context.rs

```diff
@@ -0,0 +1,72 @@
+use crate::util::rand::{FastRand, RngSeed};
+
+cfg_rt! {
+    use crate::runtime::scheduler;
+    use std::cell::RefCell;
+}
+
+struct Context {
+    /// Handle to the runtime scheduler running on the current thread.
+    #[cfg(feature = "rt")]
+    scheduler: RefCell<Option<scheduler::Handle>>,
+    rng: FastRand,
+}
+
+tokio_thread_local! {
+    static CONTEXT: Context = {
```

Comment:
Once all the thread-locals are unified here, I will consider bringing back `const` and avoiding the drop.